tee-worker/app-libs/stf/src/trusted_call.rs

```diff
@@ -251,6 +257,9 @@ where
 	// - we should return Err() if the STF execution fails
 	// - the failed top should be removed from the tool
 	// - however, the failed top hash needs to be included in the sidechain block
+	//
+	// Update2:
+	// for DI trusted calls we should return Err() to signal the errors synchronously
```

Comment:
You mean trusted calls that cannot be invoked indirectly ?

Reply:
I mean the response handling - so far it's been always returning `Ok` and put the actual result (success or failure) on the parachain